### PR TITLE
Hopefully fixes all paths here and there

### PR DESCRIPTION
**Pull request by Sebbe9123**

### CODE REVIEW

**Comment by Novacat** [comment]
random.dm is not even supposed to exist anymore, it was replaced by mob.dm in the same folder, and, should probably be deleted. Random_vr.dm should be ported over to mob_vr.dm and edited accordingly.

**Comment by Sebbe9123** [comment]
> random.dm is not even supposed to exist anymore, it was replaced by mob.dm in the same folder, and, should probably be deleted. Random_vr.dm should be ported over to mob_vr.dm and edited accordingly.

no
fucking
wonder
it felt like I had done all this already, I hate this spaghetti.

**Comment by Sebbe9123** [comment]
> Remove random.dm, fix random_vr.dm to be mob_vr.dm. Basically remove a bunch of duplicate definitions that are not necessary.

DOne, Gonna keep the branch for the next month in case something dissapeared that I didn't notice but the review thing is donnerino.

File: code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm [hunk 1]
```diff
@@ -84,7 +84,7 @@
 				update_patient()
 			return
 /* //VORESTATION AI TEMPORARY REMOVAL
-		if(istype(target, /mob/living/simple_mob/mouse)) //Edible mice, dead or alive whatever. Mostly for carcass picking you cruel bastard :v
+		if(istype(target, /mob/living/simple_mob/animal/passive/mouse)) //Edible mice, dead or alive whatever. Mostly for carcass picking you cruel bastard :v
```
Comment (Novacat) [inline code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm]: Can you test to see if that snippet of code would now work with this change? I would like to steadily fix and implement the code commented out due to the AI overhaul

Comment (Sebbe9123) [reply code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm]: boop

**Pull request merged**